gradebook/tests.py

```diff
@@ -64,7 +65,10 @@ def _get_homework_summary(self, course, attempted=False):
         return {
             u'url_name': u'Sequence_2',
             u'display_name': u'Sequence 2',
-            u'location': 'i4x://{org}/{num}/sequential/Sequence_2'.format(org=course.org, num=course.number),
+            u'location': 'block-v1:{org}+{num}+GRAD1+type@sequential+block@Sequence_2'.format(
```

Comment:
`GRAD1` is course run if you can replace that with `course.run` as you did for org and number that would make it more flexible.